### PR TITLE
106 Fixing <p> tag bug in save/edit/new for abstracts

### PR DESCRIPTION
**Pull request by rylanb**

### CODE REVIEW

File: app/decorators/proposal_decorator.rb [hunk 1]
```diff
@@ -69,7 +69,7 @@ def details
     h.markdown(object.details)
   end
 
-  def abstract
+  def abstract_markdown
```
Comment (gitcow) [inline app/decorators/proposal_decorator.rb:72]: good idea :)

File: vendor/assets/javascripts/jquery-ui-timepicker-addon.js [hunk 2]
```diff
@@ -7,7 +7,13 @@
...
  */
 
-(function ($) {
+(function (factory) {
+	if (typeof define === 'function' && define.amd) {
+		define(['jquery', 'jquery-ui'], factory);
+	} else {
+		factory(jQuery);
+	}
+}(function ($) {
```
Comment (gitcow) [inline vendor/assets/javascripts/jquery-ui-timepicker-addon.js:16]: This is interesting. So it looks like you've added some AMD module support? That's pretty neat.

Comment (rylanb) [inline vendor/assets/javascripts/jquery-ui-timepicker-addon.js:16]: Not me, I just updated the plugin. :)

File: vendor/assets/javascripts/jquery-ui-timepicker-addon.js [hunk 3]
```diff
@@ -1696,19 +1741,39 @@
...
+				 *
+				 * @todo ER1. Since altInput can have a totally difference format, convert it to standard format by reading input format from "altFormat" and "altTimeFormat" option values
+				 */
+				currDT = tp_inst.$altInput.val();
+			}
+			else {
+				currDT = tp_inst.$input.val();
+			}
+
+			if (date && tp_inst._parseTime(currDT, !inst.settings.timeOnly)) {
```
Comment (gitcow) [inline vendor/assets/javascripts/jquery-ui-timepicker-addon.js]: I'm curious to hear the backstory on this.

**Comment by gitcow** [comment]
I think my only concern with the js hacking is the risk of someone accidentally upgrading the plugin at some point and losing these changes. Perhaps a trip-wire of some sort, like a spec that asserts the version of the plugin, would be enough to let some future developer know that they've inadvertently overwritten custom changes? Beyond that, you could include a brief summary of the changes you made and stick it at the top of the file. 

Otherwise, everything looks good to me. I didn't fully grok the timezone stuff you did, but the rest of your changes seem a-OK. (thumbsup)

**Comment by rylanb** [comment]
@gitcow no JS hacking occurred. I went to: https://github.com/trentrichardson/jQuery-Timepicker-Addon and got the newest versions I could find. Copy/pasted over what was in there before.

File: vendor/assets/javascripts/jquery-ui-timepicker-addon.js (hunk 2 above)
Comment (gitcow) [inline vendor/assets/javascripts/jquery-ui-timepicker-addon.js:16]: I like to start my morning off with some assumptions

**Comment by gitcow** [comment]
@rylanb Haha yeah ok. I feel silly for thinking that. I think part of the reason is that I had to hack datetimepicker a few years ago as part of a side project, and the changes here reminded me of that, so it seemed totally plausible.

**Comment by rylanb** [comment]
Yup! I think that's a fine thought. Don't blame you for thinking it. Shoulda commented that part more thoroughly.

**Pull request merged**